app/models/account.rb

```diff
@@ -37,4 +39,8 @@ class Account < ActiveRecord::Base
   def downcase_subdomain
     self.subdomain = subdomain.try(:downcase)
   end
+
+  def single_tenant_mode?
+    Hours.single_tenant_mode?
+  end
```

Comment:
The single_tenant_mode? from Account is used in validation for subdomain - we don't need subdomain for account if app works in single tenatnt mode. Sadly Hours.single_tenant_mode? dosen't work for the validation. Do you prefer move the validation to custom validator and in the new validator use Hours.single_tenant_mode? method ?